scripts/publish.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+PARAMS=""
+while (( "$#" )); do
+  case "$1" in
+    -k|--galaxy-key)
+      KARG=$2
+      shift 2
+      ;;
+    --) # end argument parsing
+      shift
+      break
+      ;;
+    -*|--*=) # unsupported flags
+      echo "Error: Unsupported flag $1" >&2
+      echo "Please use ./publish.sh [-k <galaxy key> | --galaxy-key <galaxy key>]" >&2
+      exit 1
+      ;;
+    *) # preserve positional arguments
+      PARAMS="$PARAMS $1"
+      shift
+      ;;
+  esac
+done
+
+#
+# Initial variables
+#
+
+namespace=os_migrate
+name=os_migrate
+all_published_versions=$(curl https://galaxy.ansible.com/api/v2/collections/$namespace/$name/versions/ | jq -r '.results' | jq -c '.[].version')
+current_galaxy_version=$(cat actions/galaxy.yml | shyaml get-value version)
+current_galaxy_namespace=$(cat actions/galaxy.yml | shyaml get-value namespace)
+current_galaxy_name=$(cat actions/galaxy.yml | shyaml get-value name)
+publish="1"
+
```

Comment:
Because of issue #12 we should also check here that versions defined in galaxy.yml and const.py are the same. Simple grep/awk-based check would suffice i think. Not a blocker, i can post it subsequently myself if you want to land this first.

Reply:
Ideally we'd fix #12 but i'm not sure about a sufficiently clean solution...

Reply:
Ok i just realized the collection always needs to be built to do anything with it and get the proper Python import paths, Python imports fail otherwise. So #12 does have an acceptable solution and we can fix it, so no need to insert anything here. I'll close this comment thread.